.konflux/bundle/overlay/release.in.yaml

```diff
@@ -0,0 +1,15 @@
+---
+variables:
+  # PLACEHOLDER_ variables will be substituted by ad-hoc overlay logic
+  containerImage: PLACEHOLDER_CONTAINER_IMAGE
+  description: |
+      NUMA Resources Operator for OpenShift
+      Provides NUMA-aware scheduling and resources management for workloads.
+
+      You can find additional guidance in the upstream repository: https://github.com/openshift-kni/numaresources-operator
+  display_name: "NUMA Resources Operator"
+  manager_version: "numaresources-operator.v4.21.0"
+  # min_kube_version should match ocp
+  # https://access.redhat.com/solutions/4870701
+  min_kube_version: "1.32.0"
```

Comment:
@ffromani  up until now this was set to 1.23.0 ,  it makes sense to change it to something more up to date (although this was never blocking)
Do you see any risk here ?  cc @shajmakh